### PR TITLE
[bitnami/mongodb] remove strange mongo validation between replicacount and loadBalancerIPs

### PR DESCRIPTION
**Pull request by israelshenkar**
why mongo .Values.externalAccess.service.loadBalancerIPs needs to be the same as .Values.replicaCount ?

getting an error when trying to create replicaset:

Error: template: mongodb/templates/NOTES.txt:171:4: executing "mongodb/templates/NOTES.txt" at <include "mongodb.validateValues" .>: error calling include: template: mongodb/templates/_helpers.tpl:213:51: executing "mongodb.validateValues" at <fail>: error calling fail: 
VALUES VALIDATION:
mongodb: .Values.externalAccess.service.loadBalancerIPs
    Number of replicas and loadBalancerIPs array length must be the same.

@FraPazGal     

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by mosheavni** [comment]
I agree, it makes no sense.

**Review by Mauraza — COMMENTED** [review]
Hi @brew,
thanks for the contribution, could you do a rebase the master?

File: bitnami/mongodb/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/bitnami-docker-mongodb
   - https://mongodb.org
-version: 10.6.1
+version: 10.6.2
```
[the remark this answers is not in the document]
Comment (brew) [reply bitnami/mongodb/Chart.yaml:29]: I don't think you mean to mention me :D

**Review by juan131 — CHANGES_REQUESTED** [review]
That's only the case when external access is enabled and you're not using auto-discovery. 

The reason:

A LoadBalancer service is created per MongoDB replica. Therefore, if you have 5 replicas, 5 services will be created.
When not configuring auto-discovery, you need to provide the list of IPs in advanced for al these services and hence, the validation.

File: bitnami/mongodb/Chart.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (israelshenkar) [reply bitnami/mongodb/Chart.yaml]: updated to 10.6.5 as master is on 10.6.4 :-)

**Comment by Mauraza** [comment]
Hi @israelshenkar,

we can't remove that part, as @juan131 commented there is a case that needs it. 
Could you open an issue with the problem? we will be happy to help you to solve it.

**Pull request closed without merging**